server/core/build.gradle.kts

```diff
@@ -54,8 +54,8 @@ dependencies {
 	// and not exposed to consumers on their own compile classpath.
 	implementation("com.google.flatbuffers:flatbuffers-java:22.10.26")
 	implementation("commons-cli:commons-cli:1.5.0")
-	implementation("com.fasterxml.jackson.core:jackson-databind:2.15.1")
-	implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.15.1")
+	implementation("com.fasterxml.jackson.core:jackson-databind:2.13.5")
+	implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.13.5")
```

Comment:
Why the rollback in version?

Reply:
i think i dmed you about this, its the use of some reflection that robovm didnt support